### ISSUE TITLE
Error with macOS network sharing

### ISSUE THREAD

**Issue by leomiglio02** [utmapp/UTM#3636]
**Describe the issue**
if you try to open a vm with network and you have the internet sharing turned on in System Preferences > Sharing, you will get an error

**Configuration**
* UTM Version: 3.0.4
* OS Version: 12.2.1
* Apple Silicon

**Crash log**
Device 'vmnet-macos' could not be initialized

**Comment by leomiglio02** [comment]
I think is a qemu problem, but it can be useful have a warning if utm find that

**Comment by ktprograms** [comment]
@osy This is interesting. Do you know if Internet Sharing use vmnet.framework internally?

**Comment by osy** [comment]
No but both use the same underlying technology. I believe this is a known issue that only Apple can fix.

**Comment by ktprograms** [comment]
On Fri Feb 18, 2022 at 10:44 AM +08, osy wrote:
> No but both use the same underlying technology. I believe this is a known issue that only Apple can fix.

I see. Thanks.

**Comment by aisobe** [comment]
I can not display my shared directory after the 1st reboot. Is this thread the discussion?

**Comment by conath** [comment]
@aisobe No, you likely need to install the [guest tools](https://mac.getutm.app/support/).

**Comment by aisobe** [comment]
@conath I did that, the shared folder Z drive in Win11 Arm Preview is gone after several reboots. Why?

**Comment by conath** [comment]
I don’t know. Please create a separate issue, thanks.
